ds-caselaw-ingester/tests.py

```diff
@@ -75,11 +78,47 @@ def create_fake_bulk_file(*args, **kwargs):
     shutil.copyfile(BULK_TARBALL_PATH, "/tmp/BULK-0.tar.gz")
 
 
+@pytest.fixture
+@patch(
+    "lambda_function.Ingest.save_tar_file_in_s3",
+    return_value="/tmp/TDR-2022-DNWR.tar.gz",
+)
+def v2_ingest(fake_s3):
+    create_fake_tdr_file()
+    return lambda_function.Ingest.from_message_dict(v2_message)
+
+
+@pytest.fixture
+@patch("lambda_function.Ingest.save_tar_file_in_s3", return_value="/tmp/BULK-0.tar.gz")
+def s3_ingest(fake_s3):
+    create_fake_bulk_file()
+    return lambda_function.Ingest.from_message_dict(s3_message)
+
+
+@pytest.fixture
+@patch(
+    "lambda_function.Ingest.save_tar_file_in_s3",
+    return_value="/tmp/TDR-2022-DNWR.tar.gz",
+)
+def fcl_ingest(fake_s3):
+    "Fake a FCL reparse message (badly)"
+    new_message = copy.deepcopy(v2_message)
+    new_message["parameters"]["originator"] = "FCL"
+
+    return lambda_function.Ingest.from_message_dict(new_message)
+
+
 class TestHandler:
+
+    def test_fixture_works(self, v2_ingest, s3_ingest):
+        """We get the XML of the data and extract the URI from it successfully using the fixtures"""
+        assert v2_ingest.uri == "ewca/civ/2022/111"
+        assert s3_ingest.uri == "ukut/iac/2012/82"
+
     @patch("lambda_function.api_client", autospec=True)
     @patch("lambda_function.boto3.session.Session")
-    @patch("lambda_function.send_updated_judgment_notification")
-    @patch("lambda_function.send_new_judgment_notification")
+    @patch("lambda_function.Ingest.send_updated_judgment_notification")
+    @patch("lambda_function.Ingest.send_new_judgment_notification")
```

Comment:
Is there a way to group patches so we don't have to have so many annotations per test?